examples/example.yaml

```diff
@@ -15,13 +15,25 @@
 apiVersion: crdb.cockroachlabs.com/v1alpha1
 kind: CrdbCluster
 metadata:
-  name: crdb-example
+  name: cockroachdb
 spec:
   dataStore:
-    emptyDir: {}
+    pvc:
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
```

Comment:
This stanza is under the dataSource which is incorrect and will not work.

Reply:
Can you please clarify? This is identical to the currently published [`example.yaml`](https://github.com/cockroachdb/cockroach-operator/blob/master/examples/example.yaml).

Reply:
The resources block has the incorrect indentation